pvlib/iotools/acis.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import pandas as pd
+import numpy as np
+
+
+def get_acis_precipitation(latitude, longitude, start, end, dataset,
+                           url="https://data.rcc-acis.org/GridData", **kwargs):
```

Comment:
Is there a preference for having the URL defined as a module-level parameter vs. having it as a default value?